### DISCUSSION TITLE
[BUG] Dudas del componente documentos

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#85] [4. Bug Reports] [answered]
Hola tengo una duda sobre el componente documentos.

1.- Cuando configuro el componente y dejo como documento especifico pdf y guardo, luego en el formulario cuando selecciono un documento "pdf", me arroja que existe un error de soporte. es un bug o se debe hacer algo?

Adjunto:

1.- ss dela configuracion del fomrulario +el error
![image](https://user-images.githubusercontent.com/61419471/115059365-763dde00-9eb4-11eb-8594-5425c54c92cc.png)

![image](https://user-images.githubusercontent.com/61419471/115059552-b3a26b80-9eb4-11eb-9557-a7608d1192a4.png)

2.- Código

```{ 
    "_id" : ObjectId("5ecfbc919b891e78b96e48e3"), 
    "command" : {
        "commands" : [

        ], 
        "resetIdentifiers" : [

        ], 
        "isCommanded" : "adjuntara_oc", 
        "values" : [
            {
                "_id" : ObjectId("6079c28d36b40e00093fcb43"), 
                "op" : "eq", 
                "target" : "Si"
            }
        ]
    }, 
    "display" : [
        "Adjunte OC"
    ], 
    "code" : [
        "pdf"
    ], 
    "isActive" : true, 
    "isSystemModel" : false, 
    "contentType" : "application/vnd.cotalker.survey+file", 
    "min" : NumberInt(1), 
    "max" : NumberInt(1), 
    "hideEmpty" : false, 
    "identifier" : "oc_cliente_carga", 
    "company" : ObjectId("5eaaf0d4f999796a1563e002"), 
    "modifiedAt" : ISODate("2021-04-16T16:59:57.427+0000"), 
    "__v" : NumberInt(4), 
    "help" : null, 
    "required" : false, 
    "subtype" : null, 
    "exec" : {

    }, 
    "isReadOnly" : false
}```
Si es un bug, favor avisar

**Comment by rafaelwalter** [comment, marked as answer]
Hola @NicolasPLG


Es un bug de la app. En el seguiente release estará corrigido.

Por ahora, puedes permitir todos tipos de archivos y luego del release podrás aplicar filtro por archivos PDF.

**Reply by NicolasPLG** [reply]
Hola @rafaelwalter gracias por la info, si actualmente lo tengo para todos los tipos de archivos.

Esperaré el release :)

SAlu2
